### ISSUE TITLE
[Tabs] Demo has insufficient contrast

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#3726]
### Overview
The Tabs demo app bar text is too light against the grey to pass WCAG AA-level contrast requirements (4.5:1). The current contrast is only 2.3:1. 

The Snackbar demo text is too light against white to pass WCAG AA-level contrast requirements (4.5:1). The current contrast is only 3.6:1.

The blue star on blue background only has a contrast ratio of 2.5:1.

![simulator screen shot - iphone x - 2018-05-04 at 11 54 05](https://user-images.githubusercontent.com/1753199/39637935-8408f3cc-4f92-11e8-963a-a92ed91e04d9.png)

![simulator screen shot - iphone x - 2018-05-04 at 12 00 08](https://user-images.githubusercontent.com/1753199/39638016-d9823ffc-4f92-11e8-8bc2-9ccd9765dfbc.png)

### Reproduction steps
1. Open the catalog
1. Open Tabs &rarr; Start Demo

### Version number
55.0.1

### Operating system and device
iPhone X/iOS 11.3 (Simulator)

---
#### Internal data
- Associated internal bug: [b/117178397](http://b/117178397)

**Comment by material-mirror** [comment]
The internal issue [b/117178397](http://b/117178397) is now closed. This issue is being closed as a result.
